code/model/VirtualPage.php

```diff
@@ -510,8 +497,17 @@ public function getViewer($action) {
 	 * We can't load the content without an ID or record to copy it from.
 	 */
 	public function init(){
-		if(isset($this->record) && $this->record->ID){
-			if($this->record->VersionID != $this->failover->CopyContentFrom()->Version){
+		if (isset($this->record)) {
+			if (is_array($this->record)) {
```

Comment:
This is a bad sign - tests used to pass on the assumption `$this->record` was an object, so there is some kind of API change going on here.

I have no idea how `$this->record` was ever an object... it's always an array in SS :/